tensorflow/python/kernel_tests/conv_ops_test.py

```diff
@@ -708,7 +708,7 @@ def testConv2DOnlyBottomPadding(self):
         tensor_in_sizes=[1, 2, 3, 3],
         filter_in_sizes=[2, 2, 3, 2],
         strides=[1, 1],
-        padding=[[0, 3], [0, 0]], tol=2e-5)
+        padding=[[0, 3], [0, 0]], tol=3e-5)
```

Comment:
Since the frontend API is not enabled by default, can you revert the test change?

Reply:
Done.